romancal/datamodels/tests/test_models.py

```diff
@@ -0,0 +1,15 @@
+import asdf
+
+from romancal import datamodels
+
+
+def test_model_schemas():
+    model_classes = [
+        m for m in datamodels.__dict__.values()
+        if isinstance(m, type) and issubclass(m, datamodels.RomanDataModel)
+    ]
```

Comment:
Another way to do this would be to use the `RomanDataModel.__subclasses__()` method, though this method only returns immediate subclasses, not subclasses of subclasses.  But hopefully there are none of those.  😅

An example in `jwst`:

```
In [1]: from jwst import datamodels

In [2]: datamodels.DataModel.__subclasses__()
Out[2]: 
[jwst.datamodels.reference.ReferenceFileModel,
 jwst.datamodels.amilg.AmiLgModel,
 jwst.datamodels.asn.AsnModel,
 jwst.datamodels.combinedspec.CombinedSpecModel,
 jwst.datamodels.container.ModelContainer,
 jwst.datamodels.contrast.ContrastModel,
 jwst.datamodels.cube.CubeModel,
 jwst.datamodels.image.ImageModel,
 jwst.datamodels.extract1dimage.Extract1dImageModel,
 jwst.datamodels.gls_rampfit.GLS_RampFitModel,
 jwst.datamodels.guider.GuiderRawModel,
 jwst.datamodels.guider.GuiderCalModel,
 jwst.datamodels.ifucube.IFUCubeModel,
 jwst.datamodels.ifuimage.IFUImageModel,
 jwst.datamodels.irs2.IRS2Model,
 jwst.datamodels.level1b.Level1bModel,
 jwst.datamodels.ramp.RampModel,
 jwst.datamodels.slit.SlitDataModel,
 jwst.datamodels.slit.SlitModel,
 jwst.datamodels.multiexposure.MultiExposureModel,
 jwst.datamodels.multislit.MultiSlitModel,
 jwst.datamodels.spec.SpecModel,
 jwst.datamodels.multispec.MultiSpecModel,
 jwst.datamodels.quad.QuadModel,
 jwst.datamodels.rampfitoutput.RampFitOutputModel,
 jwst.datamodels.steppars.StepParsModel,
 jwst.datamodels.trapsfilled.TrapsFilledModel]
```

Reply:
Ooo I like that better.  We can recurse into the subclasses and look for more.